cfn.yaml

```diff
@@ -51,6 +51,8 @@ Resources:
       MemorySize: 1024
       Role: !GetAtt ExecutionRole.Arn
       Runtime: java11
+      Architectures:
+        - arm64
```

Comment:
AWS Lambda functions on ARM cost 20% less and deliver better performance - [they became available on 29th September 2021](https://aws.amazon.com/blogs/aws/aws-lambda-functions-powered-by-aws-graviton2-processor-run-your-functions-on-arm-and-get-up-to-34-better-price-performance/) (less than 2 months ago). As we're already using Java 11 and no natively-compiled dependencies, the only change required to switch over is the above config.